### PR TITLE
sanitize responses in the forms controller

### PR DESCRIPTION
**Pull request by mandric**

### CODE REVIEW

File: tests/controllers/forms.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/* jshint unused: false */
```
Comment (garethbowen) [inline tests/controllers/forms.js]: Is this necessary?

File: db.js [hunk 2]
```diff
@@ -3,6 +3,21 @@ var path = require('path'),
     nano = require('nano');
 
 var couchUrl = process.env.COUCH_URL;
+
+var sanitizeResponse = function(err, body, headers, callback) {
+  // Remove the `uri` and `statusCode` headers passed in from nano.  This
+  // could potentially leak auth information to the client.  See
+  // https://github.com/dscape/nano/blob/master/lib/nano.js#L195
+  // TODO: open issue in nano project and patch
```
Comment (garethbowen) [inline db.js]: Maybe open the issue and link it?

File: controllers/forms.js [hunk 3]
```diff
@@ -115,7 +115,7 @@ module.exports = {
         //'Content-Type': 'text/xml; charset=utf-8',
         //'Content-Disposition': 'attachment; filename="{{file}}";'
         //  .replace('{{file}}', req.params.form),
-        callback(null, body, headers);
+        db.sanitizeResponse(null, body, headers, callback);
```
Comment (garethbowen) [inline controllers/forms.js]: Clean up the commented out code while you're in there?

**Comment by garethbowen** [comment]
Added a few thoughts. Otherwise feel free to merge!

File: tests/controllers/forms.js (hunk 1 above)
Comment (mandric) [inline tests/controllers/forms.js]: Well I thought the tests were readable even if the params in the callback were unused.

File: db.js (hunk 2 above)
Comment (mandric) [inline db.js]: I agree, will open the issue and update.

File: tests/controllers/forms.js (hunk 1 above)
Comment (garethbowen) [inline tests/controllers/forms.js]: Ok, fair enough.

**Pull request merged**